ehr/resources/schemas/dbscripts/sqlserver/ehr-23.000-23.001.sql

```diff
@@ -0,0 +1,432 @@
+ALTER TABLE ehr.project ALTER COLUMN account NVARCHAR(200);
+ALTER TABLE ehr.project ALTER COLUMN inves NVARCHAR(200);
+ALTER TABLE ehr.project ALTER COLUMN avail NVARCHAR(100);
+ALTER TABLE ehr.project ALTER COLUMN title NVARCHAR(400);
+ALTER TABLE ehr.project ALTER COLUMN reqname NVARCHAR(200);
+ALTER TABLE ehr.project ALTER COLUMN contact_emails NVARCHAR(MAX);
+ALTER TABLE ehr.project ALTER COLUMN inves2 NVARCHAR(200);
+EXEC core.fn_dropifexists 'project', 'ehr', 'index', 'project_name_project';
+EXEC core.fn_dropifexists 'project', 'ehr', 'index', 'project_investigatorid_project';
+EXEC core.fn_dropifexists 'project', 'ehr', 'index', 'IDX_container_project_objectid_name';
+ALTER TABLE ehr.project ALTER COLUMN name NVARCHAR(100);
+GO
+CREATE INDEX project_investigatorid_project ON ehr.project (investigatorId ASC, project ASC) INCLUDE (name);
+CREATE INDEX project_name_project ON ehr.project (name, project);
+CREATE INDEX IDX_container_project_objectid_name ON ehr.project (container, project, objectid, name);
+ALTER TABLE ehr.project ALTER COLUMN use_category NVARCHAR(100);
+ALTER TABLE ehr.project ALTER COLUMN shortname NVARCHAR(200);
+ALTER TABLE ehr.project ALTER COLUMN projecttype NVARCHAR(100);
+GO
+
+EXEC core.fn_dropifexists 'protocol', 'ehr', 'index', 'IDX_protocol_container_protocol';
+ALTER TABLE ehr.protocol ALTER COLUMN protocol NVARCHAR(200) NOT NULL;
+GO
+CREATE INDEX IDX_protocol_container_protocol ON ehr.protocol (container, protocol);
+ALTER TABLE ehr.protocol ALTER COLUMN inves NVARCHAR(200);
+ALTER TABLE ehr.protocol ALTER COLUMN title NVARCHAR(1000);
+ALTER TABLE ehr.protocol ALTER COLUMN usda_level NVARCHAR(100);
+ALTER TABLE ehr.protocol ALTER COLUMN external_id NVARCHAR(200);
+ALTER TABLE ehr.protocol ALTER COLUMN project_type NVARCHAR(200);
+ALTER TABLE ehr.protocol ALTER COLUMN ibc_approval_num NVARCHAR(200);
+ALTER TABLE ehr.protocol ALTER COLUMN contacts NVARCHAR(200);
+GO
+
+ALTER TABLE ehr.protocol_amendments ALTER COLUMN protocol NVARCHAR(200);
+ALTER TABLE ehr.protocol_amendments ALTER COLUMN Comment NVARCHAR(MAX);
+GO
+
+ALTER TABLE ehr.protocol_counts ALTER COLUMN protocol NVARCHAR(200);
+ALTER TABLE ehr.protocol_counts ALTER COLUMN species NVARCHAR(200) NOT NULL;
+ALTER TABLE ehr.protocol_counts ALTER COLUMN gender NVARCHAR(100);
+ALTER TABLE ehr.protocol_counts ALTER COLUMN description NVARCHAR(MAX);
+GO
+
+ALTER TABLE ehr.protocolexemptions ALTER COLUMN protocol NVARCHAR(100);
+ALTER TABLE ehr.protocolexemptions ALTER COLUMN exemption NVARCHAR(200);
+ALTER TABLE ehr.protocolexemptions ALTER COLUMN remark NVARCHAR(MAX);
+GO
+
+ALTER TABLE ehr.protocolprocedures ALTER COLUMN protocol NVARCHAR(200) NOT NULL;
+ALTER TABLE ehr.protocolprocedures ALTER COLUMN procedurename NVARCHAR(200);
+ALTER TABLE ehr.protocolprocedures ALTER COLUMN code NVARCHAR(100);
+ALTER TABLE ehr.protocolprocedures ALTER COLUMN frequency NVARCHAR(2000);
```

Comment:
This was originally a VARCHAR(200)
```suggestion
ALTER TABLE ehr.protocolprocedures ALTER COLUMN frequency NVARCHAR(200);
```